docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-238.adoc

```diff
@@ -0,0 +1,73 @@
+
+== Azure Cognitive Services account is not configured with managed identity
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/CognitiveServicesConfigureIdentity.py[CKV_AZURE_238]
+
+|Severity
+|HIGH
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan,ARM
+
+|===
+
+=== Description
+
+This policy ensures that the Cognitive Services account is not configured with managed identity. Managed identity is often used to securely manage credentials for a service and also to simplify secret management. However, using it with a Cognitive Services account can open up potential security risks. It can allow unauthorized services to access and manipulate sensitive information in the Cognitive Services account without necessarily requiring explicit permission. Thus, it's important not to configure a Cognitive Services account with a managed identity to prevent the probability of security threats and data breaches.
```

Comment:
This policy checks whether Cognitive Services accounts leverage managed identity. Managed identity simplifies credential management but can expose security risks in this context. It could grant unauthorized services access to sensitive information within your Cognitive Services account without necessarily requiring explicit permission. Disabling managed identity strengthens security by preventing this potential vulnerability.